src/engine/source/kvdb/src/kvdbManager.cpp

```diff
@@ -370,6 +375,7 @@ std::optional<base::Error> KVDBManager::deleteDB(const std::string& name)
         if (handler.use_count() == MAX_USE_COUNT)
         {
             m_dbs.erase(name);
+            Metrics::instance().addUpDownCounterValue("Kvdb.DatabaseInUseCounter", -1L);
```

Comment:
nitpick: Could you put a brief comment above the metric to know what it measures?